Configuration/HLT/python/autoCondHLT.py

```diff
@@ -18,15 +18,24 @@
 l1Menus= {
     'Fake'         : ( ','.join( [ 'L1GtTriggerMenu_L1Menu_Collisions2012_v3_mc'             , l1MenuRecord,connectionString, l1MenuLabel, "2015-10-26 12:00:00.000"] ), ),
     'Fake1'        : ( ','.join( [ 'L1Menu_Collisions2015_25nsStage1_v5'                     , l1MenuRecord,connectionString, l1MenuLabel, "2015-10-26 12:00:00.000"] ), ),
-    'Fake2'        : ( ','.join( [ 'L1Menu_Collisions2017_dev_r5_m4_patch_xml'                         ,l1tMenuRecord,connectionString,l1tMenuLabel, "2017-05-19 10:21:02.000"] ), ),
-    'FULL'         : ( ','.join( [ 'L1Menu_Collisions2017_dev_r5_m4_patch_xml'                         ,l1tMenuRecord,connectionString,l1tMenuLabel, "2017-05-19 10:21:02.000"] ), ),
-    'GRun'         : ( ','.join( [ 'L1Menu_Collisions2017_dev_r5_m4_patch_xml'                         ,l1tMenuRecord,connectionString,l1tMenuLabel, "2017-05-19 10:21:02.000"] ), ),
+    'Fake2'        : ( ','.join( [ 'L1Menu_Collisions2017_dev_r5_m4_patch_xml'               ,l1tMenuRecord,connectionString,l1tMenuLabel, "2017-05-19 10:21:02.000"] ), ),
+    'FULL'         : ( ','.join( [ 'L1Menu_Collisions2017_dev_r5_m4_patch_xml'               ,l1tMenuRecord,connectionString,l1tMenuLabel, "2017-05-19 10:21:02.000"] ), ),
+    'GRun'         : ( ','.join( [ 'L1Menu_Collisions2017_dev_r5_m4_patch_xml'               ,l1tMenuRecord,connectionString,l1tMenuLabel, "2017-05-19 10:21:02.000"] ), ),
     'GRun2016'     : ( ','.join( [ 'L1Menu_Collisions2016_v9_m2_xml'                         ,l1tMenuRecord,connectionString,l1tMenuLabel, "2016-10-06 19:36:53.000"] ), ),
     'HIon'         : ( ','.join( [ 'L1Menu_CollisionsHeavyIons2015_v5_uGT_xml'               ,l1tMenuRecord,connectionString,l1tMenuLabel, "2016-03-04 15:00:00.000"] ), ),
     'PIon'         : ( ','.join( [ 'L1Menu_HeavyIons2016_v3_m2_xml'                          ,l1tMenuRecord,connectionString,l1tMenuLabel, "2016-11-22 11:11:00.000"] ), ),
     'PRef'         : ( ','.join( [ 'L1Menu_Collisions2015_5TeV_pp_reference_v5_uGT_v2_mc_xml',l1tMenuRecord,connectionString,l1tMenuLabel, "2016-03-04 15:00:00.000"] ), ),
 }
 
+pfRecord = 'GBRDWrapperRcd'
```

Comment:
These lines set the L1T menus for the TSG GTs used in our TSG tests (which change more frequently). Eventually when we update the relval2017 menu the L1T menu needs to be in the actual GT key of AlCa's autoCond.py. But this is like this since many years!